theano/tensor/elemwise.py

```diff
@@ -890,14 +891,15 @@ def perform(self, node, inputs, output_storage):
             # numpy the first (faster) version leads to segfaults
             if self.ufunc:
                 ufunc = self.ufunc
+            elif not hasattr(node.tag, 'ufunc'):
+                # It happen that make_thunk isn't called, like in
+                # get_scalar_constant_value
+                self.prepare_node(node, None, None, 'py')
+                if self.ufunc:
+                    ufunc = self.ufunc
```

Comment:
We already checked self.ufunc above.  Also, calling perform without prepare_node/make_thunk should not be supported.  We should fix get_scalar_constant_value instead.